tools/cli/go-whisk-cli/commands/sdk.go

```diff
@@ -99,6 +106,58 @@ var sdkInstallCmd = &cobra.Command{
     },
 }
 
+// addToBash() attempts to find user's .bashrc and append the line
+// "eval \"`wsk sdk install bashauto`\"" to it unless the line already exists
+func addToBash() error {
+    var usr *user.User
+    var bash *os.File
+    var exists bool
+    var err error
+
+    bashCheck := "wsk sdk install bashauto"
+    bashCmd := "eval \"`wsk sdk install bashauto`\""
+
+    if usr, err = user.Current(); err != nil {
+        whisk.Debug(whisk.DbgError, "Couldn't find User Directory: %s\n", err)
+        return err
+    }
+    whisk.Debug(whisk.DbgInfo, "Found current user %s\n", usr.HomeDir)
+    bashLoc := fmt.Sprintf("%s/%s", usr.HomeDir, ".bashrc")
```

Comment:
Can you provide any insight into this command `homedir.Expand(".bashrc")`. I have only been able to get it to work by changing it to the following `homedir.Expand("~/.bashrc")`.